lib/manager/gitlabci/extract.ts

```diff
@@ -61,7 +61,7 @@ export function extractPackageFile(content: string): PackageFile | null {
           foundImage = false;
           const serviceImageLine = skipCommentLines(lines, lineNumber + 1);
           logger.trace(`serviceImageLine: "${serviceImageLine.line}"`);
-          const serviceImageMatch = /^\s*-\s*'?"?([^\s'"]+)'?"?\s*$/.exec(
+          const serviceImageMatch = /^\s*-\s*(?:name:*\s*)*'?"?([^\s'"]+)'?"?\s*$/.exec(
```

Comment:
This regex looks pretty ugly to me.

Reply:
```suggestion
          const serviceImageMatch = /^\s*-\s*(?:name:\s*)?'?"?([^\s'"]+)'?"?\s*$/.exec(
```

I think the stars are to much and would allow `name::::::::::: image` or `name    image` or `name: name: image`